back-end/schema/projects-schema.js

```diff
@@ -0,0 +1,24 @@
+import { ObjectID } from "bson";
+import mongoose from "mongoose";
+
+
+// This defines the structure of the projects data object
+const projectSchema = new mongoose.Schema({
+    project_name: {
+        type: String,
+        required: true,
+    },
+    project_id: {
+        type: ObjectID,
+        required: true,
+    },
+    api_key: {
+        type: String,
+        required: true
+    },
+    transcripts: {
+        type: Array
+    }
+});
+
+export const Projects = mongoose.model("Projects", projectSchema);
```

Comment:
Nice work with the schema! This looks great and will be very helpful to make sure were adding data with the right format